src/pretix/base/management/commands/shell_scoped.py

```diff
@@ -18,6 +18,7 @@ def handle(self, *args, **options):
             cmd = 'shell_plus'
         except ImportError:
             cmd = 'shell'
+            del options['skip_checks']
```

Comment:
alternatively raise and expect `django_extensions` to be installed/add to requirements-dev